### ISSUE TITLE
这个测试精度。。。

### ISSUE THREAD

**Issue by PanFei748** [liuch37/sar-pytorch#6]
是不是错了啊

**Comment by liuch37** [comment]
Note text recognition requires very large amount of data to have good accuracy. In my experiments I only use the single original small dataset for training. Therefore you see poor performance. But you can see from there if increasing number of training data, the accuracy increases. These experiments are just for sanity check.
